aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -5,110 +5,175 @@ description: Learn how to host an ASP.NET Core app in a Windows Service.
 
 ## Get started
 
-This section explains the minimum changes required to set up an existing ASP.NET Core project to run in a service.
+The following minimum changes are required to set up an existing ASP.NET Core project to run in a service:
 
-1. Install the NuGet package [Microsoft.AspNetCore.Hosting.WindowsServices](https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting.WindowsServices/).
+1. Add a package reference for [Microsoft.AspNetCore.Hosting.WindowsServices](https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting.WindowsServices/).
 
-2. Make the following changes in `Program.Main`:
+1. Make the following changes in `Program.Main`:
 
-   * Call `host.RunAsService` instead of `host.Run`.
+   * Call [host.RunAsService](/dotnet/api/microsoft.aspnetcore.hosting.windowsservices.webhostwindowsserviceextensions.runasservice) instead of `host.Run`.
 
-   * If the code calls `UseContentRoot`, use a path to the publish location instead of `Directory.GetCurrentDirectory()`.
+   * If the code calls `UseContentRoot`, use a path to the app's published location instead of `Directory.GetCurrentDirectory()`.
 
-   # [ASP.NET Core 2.x](#tab/aspnetcore2x/)
+     ::: moniker range=">= aspnetcore-2.0"
 
-   [!code-csharp[](windows-service/sample/Program.cs?name=ServiceOnly&highlight=3-4,7,12)]
+     [!code-csharp[](windows-service/sample/Program.cs?name=ServiceOnly&highlight=3-4,7,11)]
 
-   # [ASP.NET Core 1.x](#tab/aspnetcore1x/)
+     ::: moniker-end
 
-   [!code-csharp[](windows-service/sample_snapshot/Program.cs?name=ServiceOnly&highlight=3-4,8,14)]
+     ::: moniker range="< aspnetcore-2.0"
 
-   ---
+     [!code-csharp[](windows-service/sample_snapshot/Program.cs?name=ServiceOnly&highlight=3-4,8,13)]
 
-3. Publish the app to a folder. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles) that publishes to a folder.
+     ::: moniker-end
 
-4. Test by creating and starting the service.
+1. Publish the app to a folder. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles) that publishes to a folder.
 
-   Open a command shell with administrative privileges to use the [sc.exe](https://technet.microsoft.com/library/bb490995) command-line tool to create and start a service. If the service is named MyService, published to `c:\svc`, and named AspNetCoreService, the commands are:
+   To publish the sample app from the command line, run the following command in a console window from the project folder:
+
+   ```console
+   dotnet publish --configuration Release --output c:\\svc
+   ```
+
+   The double-backslash escapes a backslash in the command-line argument for the output path. The project is published to the *c:\\svc* folder.
```

Comment:
Where is this needed? I don't need double slashes on cmd.

Reply:
I must have cut-and-paste from Line 59!

*I really **DO** need a vacation!* 🏖